### PR TITLE
feat: create result page

### PR DESCRIPTION
**Pull request by blertaazemi**

### CODE REVIEW

**Review by ArianitSGashi — APPROVED** [review]

File: src/OnlineVotingS.API/Models/CompleteElectionViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Models/CompleteElectionViewModel.cs]: remove white line

File: src/OnlineVotingS.API/Controllers/AdminController.cs [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
+        public IActionResult Dashboard()
+        {
+            return View();
+        }
+
+        public IActionResult Voter()
+        {
+            return View();
+        }
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Controllers/AdminController.cs]: remove this empty line

File: src/OnlineVotingS.API/Controllers/ElectionController.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using OnlineVotingS.API.Models;
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Controllers/ElectionController.cs]: remove extra line

File: src/OnlineVotingS.API/Controllers/ElectionController.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            };
+
+            var model = new CompleteElectionViewModel
+            {
+                OngoingElections = ongoingElections
+            };
+
+            return View("~/Views/Admin/Election/CompleteElection.cshtml", model);
+        }
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.API/Controllers/ElectionController.cs]: remove extra line

**Pull request merged**